src/roles/vnsutil-predeploy/tasks/vcenter.yml

```diff
@@ -22,6 +22,7 @@
       tasks_from: configure-ovftool
     vars:
       management_network: "Management"
+      vmfolder: "{{ vcenter.vmfolder | default(omit) }}"
```

Comment:
configure-ovftool does not use a variable named "vmfolder". This line has no effect. configure-ovftool uses vcenter.vmfolder, which is a different variable in a different scope. Please remove vmfolder from here.